eQuiz/modules/eQuiz.Web/Areas/Admin/Scripts/review/start-from.filter.js

```diff
@@ -0,0 +1,9 @@
+﻿(function(angular) {
+    angular.module('equizModule').filter('startFrom', function () {

```

Comment:
Wrong syntax of filter declaration.
Use quizReview-data.service.js as an example